### PR TITLE
Bump root dependencies during Talos execution

### LINKED ISSUE #1753: Talos - Bump dependancies in base level package.json
**Is your feature request related to a problem? Please describe.**
Currently talos only updates deps in the packages folder, it also needs to be able to update them in the top level.

**Describe the solution you'd like**
Allow for bumping deps in the base level package lock, increment its version and also update changelog

**Additional context**

No testing effort required from a tester.

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #1753 

**Overall change:** 
Change scripts to include root package to list of packages

**Code changes:**
- Added root packages to all stages of talks pipeline
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: scripts/publish/test/getPackages.test.js [hunk 1]
```diff
@@ -4,31 +4,36 @@ describe(`Publish Script - getPackages`, () => {
     jest.resetModules();
   });
 
-  it('returns array of packages with two packages found', () => {
+  it('returns array of packages with the root package and two other packages found', () => {
```
Comment (dr3) [inline scripts/publish/test/getPackages.test.js]: Just realised.. this test isnt testing the correct file... `scripts/publish/test/getPackages.test.js` doesnt test `scripts/publish/src/getPackages.js` and infact nothing uses it :L 

Can you delete `scripts/publish/src/getPackages.js` please, and combine `scripts/publish/test/getPackages.test.js` and `scripts/utilities/getPackages/index.test.js` please

**Review by dr3 — CHANGES_REQUESTED** [review]
Just one thing which I noticed :P Otherwise LGTM

File: scripts/publish/test/getPackages.test.js (hunk 1 above)
Comment (pharingee) [reply scripts/publish/test/getPackages.test.js]: Ok.

**Review by dr3 — APPROVED** [review]

**Comment by dr3** [comment]
@pharingee this has 78% test coverage according to codeclimate, any ideas where its lacking?

**Comment by pharingee** [comment]
Not really. I think testing `parseUpgradedPackages` for the base package might be the only test missing here. I'll add that to it.

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**